1.pre_process_data/scripts/0.fix_pathing.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook fixes issues with data paths by replacing spaces with underscores
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
```

Comment:
I didn't see that Pandas was needed in this notebook, consider removing it.

Reply:
removed